Gemfile

```diff
@@ -9,6 +9,4 @@ gemspec
 gem 'bump', require: false
 gem 'rake'
 gem 'rspec'
-gem 'rubocop', github: 'rubocop-hq/rubocop'
```

Comment:
Please keep it as this gem is catching up and dogfooding to edge RuboCop core.

Reply:
For what? The current version of `rubocop-performance` can be incompatible with the current version of `rubocop` and CI can not catch this, or catch phantom errors.

Reply:
Let's update version locks after new releases, maybe via bots like Dependabot or Depfu.

Reply:
Yeah, CI isn't well prepared, but it's hard to keep up with the changes while waiting for the release. I like this option for maintenance.

Reply:
Maybe it should be commented line of code and anybody can "enable" it when they want?

The first, there is CI, as I wrote. The second, imagine a newcomer which want to contribute small changes but facing fails because of this unstable dependencies and becomes sad or even gives up.